nvflare/lighter/impl/static_file.py

```diff
@@ -860,14 +873,17 @@ def finalize(self, project: Project, ctx: ProvisionContext):
                     temp_section=TemplateSectionKey.COMM_CONFIG,
                     file_name=ProvFileName.COMM_CONFIG,
                     replacement=replacement_dict,
-                    content_modify_cb=self._remove_undefined_port,
+                    content_modify_cb=_remove_undefined_port,
                 )
 
         # create start_all.sh
         self._create_start_all(project, ctx)
 
     @staticmethod
-    def _create_start_all(project: Project, ctx: ProvisionContext):
+    def _append(content: str, participant: Participant) -> str:
+        return content + f"./{participant.name}/startup/start.sh\n"
+
+    def _create_start_all(self, project: Project, ctx: ProvisionContext):
```

Comment:
since _append is a staticmethod, this method can still remain static if we want to, we can call "StaticFileBuilder._append"